### ISSUE TITLE
talkback speaks in the middle of calls

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#191]
```
What steps will reproduce the problem?
1. with talkback running, start a voice call with someone with the phone next 
your ear and the screen automatically turned off.
2. Recieve an incoming text message that causes a notification to be displayed 
on the screen.
3. Talkback reads out the entire text message overriding the speech of the 
person you are talking with and there is no way to stop talkback until it has 
finished reading the message text.

4. An alternative flow is for an incoming call, if you pick the call as soon as 
talkback has started reading out the incoming caller's contact information, 
talkback doesn't stop reading the information and the speech of the caller is 
drowned out in talkback speech in your ear.

What is the expected output? What do you see instead?
1.  The expected output is that talkback should not speak at all during an 
ongoing call unless if I have the screen turned on and am trying to do 
something with explore by touch or other gestures.
2. For incoming messages etc., there can be earcons that are played.
3. For a waiting call, there should be a gesture that causes talkback to speak 
the caller id in the middle of another call.


What version of the product are you using? On what operating system?
talkback 3.3.0 b3 r88 on nexus 4 running android 4.2.2


Please provide any additional information below.


```

Original issue reported on code.google.com by `manis...@gmail.com` on 28 Feb 2013 at 4:56

**Comment by GoogleCodeExporter** [comment]
```
Unable to reproduce. Are you using a third-party text-to-speech engine?
```

Original comment by `alanv@google.com` on 6 Mar 2013 at 1:22

**Comment by GoogleCodeExporter** [comment]
```
Yes, I am using Ivona tts. I'll try with Google tts and report back.
```

Original comment by `manis...@gmail.com` on 6 Mar 2013 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
Marking as resolved. Please re-open if you're still experiencing this issue 
when using Google TTS.
```

Original comment by `alanv@google.com` on 2 Apr 2013 at 10:51
- Changed state: **WontFix**
